docs/en-US/component/image.md

```diff
@@ -92,11 +92,12 @@ image/image-preview
 
 ### Image Slots
 
-| Name        | Description                                              |
-| ----------- | -------------------------------------------------------- |
-| placeholder | custom placeholder content when image hasn't loaded yet. |
-| error       | custom image load failed content.                        |
-| viewer      | custom content when image preview.                       |
+| Name              | Description                                              |
+| ----------------- | -------------------------------------------------------- |
+| placeholder       | custom placeholder content when image hasn't loaded yet. |
+| error             | custom image load failed content.                        |
+| viewer            | custom content when image preview.                       |
+| progress ^(2.9.2) | custom progress content when image preview.              |
```

Comment:
refer https://element-plus.org/en-US/component/badge.html#slots, Add a type column to indicate the type of parameter received.